docs/populating-configuration.md

```diff
@@ -0,0 +1,353 @@
+> NOTE some step in this document might refer to Red Hat internal components
+  that you do not have access to
+
+# Populating Configuration
+The document describes how to prepare Fleet Manager to be able to start by
+populating its configurations.
+
+For convenience, the following Makefile target is provided:
+
+```
+make secrets/setup/empty
+```
+
+This target initializes empty files that are necessary for the service to boot.
+It can be used to quickly get the service running locally, although it does not
+provide a functional kas-fleet-manager.
+
+Follow the subsections to find information about how to configure these files properly.
+
+## Interacting with the Fleet Manager API
+
+See the [Interacting with Fleet Manager](interacting-with-fleet-manager.md)
+document.
+
+## Setting up OCM tokens
+
+The Fleet Manager itself requires the use of an OCM token so it can
+interact with OCM to perform management of Data Plane clusters.
+
+In order for the Fleet Manager to do so, an OCM offline token should be configured.
+To do so, retrieve your OCM offline token and then configure it for Fleet
+Manager by running:
+```
+make ocm/setup OCM_OFFLINE_TOKEN=<your-retrieved-ocm-offline-token>
+```
+
+> NOTE: Information about how to retrieve an OCM token can be found in the
+  [Interacting with Fleet Manager API](interacting-with-fleet-manager.md)
+  document
+
+## Allowing creation of *Standard* Kafka instances
+
+Fleet Manager is able to create two types of Kafka instances:
+* Developer instances
+  * Instances of this type are automatically deleted after 48 hours by default
+    > NOTE: This can be controlled by setting the `lifespanSeconds` attribute
+      in the `config/kafka-instance-types-configuration.yaml` configuration file
+      for developer instances
+  * All authenticated users that make use of the Fleet Manager can
+    request the creation of a Kafka developer instance
+  * There is a limit of one instance per user
```

Comment:
note that this is by default and can actually be changed via configuration: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/internal/kafka/internal/environments/development.go#L40

Reply:
reworded